src/model/client/AbstractClient.ts

```diff
@@ -93,18 +93,18 @@ export abstract class AbstractClient {
       let wrappedResponse: EndpointResponse<E> = response.data[property];
 
       // If the response is paginated, wrap it.
-      if (response.data.page || response.data.size || response.data.results) {
+      if (response.data.page !== undefined || response.data.size !== undefined || response.data.results !== undefined) {
```

Comment:
leest het niet stukken fijner om hier losse `isNotDefined` functions van te maken OF destructure deze values.

Reply:
zeker